api/managementpb/ia/alerts.proto

```diff
@@ -0,0 +1,81 @@
+syntax = "proto3";
+
+package ia.v1beta1;
+
+option go_package = "api/managementpb/ia;iav1beta1";
+
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+import "managementpb/ia/boolean_flag.proto";
+import "managementpb/ia/status.proto";
+import "managementpb/severity.proto";
+
+// TODO move it to parent directory / package once API is v1-stable
+
+// Alert represents Alert.
+message Alert {
+  // ID.
+  string alert_id = 1;
```

Comment:
Can it be simple `id`?

Reply:
It could be, but we use foo_id to differentiate between different IDs in the same message when they are present, and to avoid problems in Go code